app/domain/errors.rb

```diff
@@ -514,6 +514,34 @@ module AuthnJwt
         msg: "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
         code: "CONJ00108E"
       )
+
+      MappedClaimsMissingInput = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, mapped claims value is empty, or was not found.",
```

Comment:
@hilagross @shulifink 
Please review

Reply:
Failed to parse mapped claims, **the** value is empty, or was not found

I think it was duplicated